src/bp/ui-studio/src/web/views/FlowBuilder/diagram/index.jsx

```diff
@@ -38,6 +40,24 @@ export default class FlowBuilder extends Component {
     this.diagramEngine.registerLinkFactory(new DeletableLinkFactory())
 
     this.setModel()
+
+    window.highlightNode = (flow, node) => {
```

Comment:
let's stop poluting the `window` object.  I'd rather have a static method on this class (`FlowBuilder.highlightNode`) if we want to keep it simple (no redux) and cleanier.  The FlowBuilder instance can override the static method at onComponentDidMount() time (and unbind itself at unmount)

But feels like Redux would be cleaner here

Reply:
The calling method is inside an iframe (channel web container), so there's not a lot of options to communicate.